boards/sim/sim/sim/configs/rpserver/defconfig

```diff
@@ -54,7 +54,6 @@ CONFIG_SCHED_CHILD_STATUS=y
 CONFIG_SCHED_HAVE_PARENT=y
 CONFIG_SCHED_HPWORK=y
 CONFIG_SCHED_WAITPID=y
-CONFIG_SIM_M32=y
```

Comment:
@yamt can we support 32bit on macOS? since OpenAMP just support 32bit address space now, rpproxy/rpserver stop work without CONFIG_SIM_M32.

Reply:
no. the recent versions of macOS are 64-bit only.

Reply:
But the 64bit rpproxy/rpserver will crash directly, so it's better to skip some build on macOS, maybe enhance datalist format like this:
sim:rpproxy,!macOS

Reply:
maybe we can just preprocess the list with m4 or cpp or whatever.